public/views/partials/molecules/status-label.liquid

```diff
@@ -0,0 +1,83 @@
+---
+metadata:
+  name: Status label
+  params:
+    severity:
+      - info
+      - error
+      - success
+      - warning
+    icon:
+      - true
+      - false
+    alignment:
+      - left
+      - right
+    content: Fill your profile
+  styleguide:
+    - severity: info
+      content: Draft
+      icon: true
+      alignment: left
+    - severity: error
+      content: Closed
+      icon: true
+      alignment: right
+    - severity: success
+      content: <b>Open</b> until 20 July 2021
+      icon: true
+    - severity: warning
+      content: Under Evaluation
+      icon: true
+    - severity: info
+      content: Draft
+      icon: false
+    - severity: error
+      content: Closed
+      icon: false
+      alignment: left
+    - severity: success
+      content: <b>Open</b> until 20 July 2021
+      icon: false
+    - severity: warning
+      content: Under Evaluation
+      icon: false
+---
+{% parse_json icons %}
+{
+  "success": { "name": "checkBadge", "color": "text-confirmation", "bgcolor": "bg-confirmation" },
+  "error": { "name": "warningCloud", "color": "text-important", "bgcolor": "bg-important" },
+  "info": { "name": "info", "color": "text-graphic", "bgcolor": "bg-graphic" },
+  "warning": { "name": "warning", "color": "text-warning", "bgcolor": "bg-warning" }
+}
+{% endparse_json %}
+{% liquid
+  assign icon = params.icon | default: false
+  assign alignment = params.alignment | default: 'right'
+  assign severity = params.severity | default: 'info'
+  assign content = params.content
+
+  assign classes = 'px-2 py-1 h-6 rounded flex items-center align-center border border-divider bg-panel '
+
+  if alignment == "left"
+    assign classes = classes | append:  " flex-row-reverse"
+    assign content_classes = "mr-1"
+  else
+    assign content_classes = "ml-1"
+  endif
+
+  assign icon_classes = 'w-3 h-3 ' | append: icons[severity].color
+  assign icon_params = null | hash_merge: name: icons[severity].name, class: icon_classes
+%}
+<div class="inline-block">
```

Comment:
Please add support for additional classes for the component wrapper.
Also, every component should have its own css class to support easy customizations. 
(I'll add these topics to the CL documentation)

Reply:
@nemvalid fixed